### ISSUE TITLE
Generates .Rd files with line length >80 in some cases

### ISSUE THREAD

**Issue by multimeric** [r-lib/roxygen2#1457]
I have created a repository that captures a minimal example of this: https://github.com/multimeric/RoxygenLineLength. This repository already has R source files and the resulting .Rd docs produced from them, but you can also regenerate the .Rd using `roxygen2::roxygenise()` if you clone the repository.

The two cases I can consistently identify that trigger this are:
* A title section with a line length very close to 80: https://github.com/multimeric/RoxygenLineLength/blob/835a32205c9c01c7b954e6680daff9daf8ec30af/R/doc.R#L8. This seems to happen because Roxygen2 formats `title` as:
  ```
  \title{content}
  ```
  Instead of the following format that it uses for other fields, which is immune to excessive line lengths if the source documentation is also <80 chars:
  ```
  \title{
  content
  }
  ```
* Basically any use of `inheritDotParams`: https://github.com/multimeric/RoxygenLineLength/blob/835a32205c9c01c7b954e6680daff9daf8ec30af/R/doc.R#L10

**Comment by hadley** [comment]
Now closing because we've made no progress on this and it doesn't seem to affect many people.
